school/app/views/courses/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<a href='/courses/new'>Create New Course</a>
+
+<% @course.each do |course| %>
+<div>
+    <h4><%= course.name %></h4>
+    Location: <%= course.location %>
+    <p><%= course.description %></p>
+    <a href='/courses/<%= course.id %>'>Edit</a>
+    <a href='/courses/delete?id=<%= course.id %>'>Delete</a>
+    <a href='/courses/show?id=<%= course.id %>'>Show</a>
```

Comment:
Based on the feedback I gave you in your routes.rb file. 
You can now change the links on your form. If you type in `rails routes` you'll see all your possible routes. The [docs](https://guides.rubyonrails.org/layouts_and_rendering.html) has more detailed explanation.